packages/spatial/src/physics/components/ColliderComponent.tsx

```diff
@@ -51,6 +52,7 @@ export const ColliderComponent = defineComponent({
     collisionMask: S.Number(DefaultCollisionMask),
 
     //shape specific parameters
+    matchMesh: S.Bool(false),
```

Comment:
is this option necessary? we should try to keep the collider component adhering the to proposed GLTF spec

Reply:
this is being used to drive the collider based on mesh data rather than manual input, and disables the related fields in the node editor. It is used by the GLTFComponent for applying colliders